conda/recipes/libraft/meta.yaml

```
@@ -128,30 +177,52 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
     requirements:
       build:
         - {{ compiler('c') }}
-        - {{ compiler('cuda') }} {{ cuda_version }}
         - {{ compiler('cxx') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }} {{ cuda_version }}
+        {% else %}
+        - {{ compiler('cuda') }}
+        {% endif %}
+        - cuda-version ={{ cuda_version }}
         - cmake {{ cmake_version }}
         - ninja
         - sysroot_{{ target_platform }} {{ sysroot_version }}
       host:
         - {{ pin_subpackage('libraft', exact=True) }}
-        - cudatoolkit {{ cuda_version }}
-        - cuda-profiler-api {{ cuda_profiler_api_host_version }}
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        - cuda-profiler-api {{ cuda11_cuda_profiler_api_run_version }}
+        - libcublas {{ cuda11_libcublas_host_version }}
+        - libcublas-dev {{ cuda11_libcublas_host_version }}
+        - libcurand {{ cuda11_libcurand_host_version }}
+        - libcurand-dev {{ cuda11_libcurand_host_version }}
+        - libcusolver {{ cuda11_libcusolver_host_version }}
+        - libcusolver-dev {{ cuda11_libcusolver_host_version }}
+        - libcusparse {{ cuda11_libcusparse_host_version }}
+        - libcusparse-dev {{ cuda11_libcusparse_host_version }}
+        {% else %}
+        - cuda-profiler-api
+        - libcublas-dev
+        - libcurand-dev
+        - libcusolver-dev
+        - libcusparse-dev
+        {% endif %}
+        - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
```

`pin_compatible` can only be used in `run` or `test` sections. https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#extra-jinja2
```suggestion
        - cuda-version ={{ cuda_version }}
```